website/blog/2024-05-13-thallium.md

```diff
@@ -0,0 +1,326 @@
+---
+author: Tomasz Godzik
+title: Metals v1.3.1 - Thallium
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v1.3.1, which brings in a couple
+of useful improvements.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">70</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">60</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">27</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">3</td>
+  </tr>
+</tbody>
+</table>
+
+For full details:
+[https://github.com/scalameta/metals/milestone/67?closed=1](https://github.com/scalameta/metals/milestone/67?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs, Zed,
+Helix and Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Resolve classpath lazily in Bazel](#resolve-classpath-lazily-in-bazel)
+- [Replace `slowTask` with LSP progress](#replace-slowtask-with-lsp-progress)
+- [No workspace support](#no-workspace-support)
+
+## Resolve classpath lazily in Bazel
+
+Metals uses [Bazel BSP](https://github.com/jetBrains/bazel-bsp/) to interact if
+Bazel itself via the Build Server Protocol. Up until this release, we would
```

Comment:
Something is wrong with this sentence.

Reply:
it should have been `with` instead of `if`. Changed!